README.rst

```diff
@@ -104,15 +104,21 @@ To build Lightning plugins from source you can run
 
     $ PL_BACKEND=${PL_BACKEND} pip install pybind11 pennylane-lightning --no-binary :all:
 
-where ``${PL_BACKEND}`` can be ``lightning_qubit`` (default), ``lightning_gpu`` or ``lightning_kokkos``.
+where ``${PL_BACKEND}`` can be 
+
+- ``lightning_qubit`` (default), 
+- ``lightning_gpu`` 
+- ``lightning_kokkos``
+- ``lightning_tensor``
+
 The `pybind11 <https://pybind11.readthedocs.io/en/stable/>`_ library is required to bind the C++ functionality to Python.
 
 A C++ compiler such as ``g++``, ``clang++``, or ``MSVC`` is required.
 On Debian-based systems, this can be installed via ``apt``:
 
 .. code-block:: console
 
-    $ sudo apt -y update &&
+    $ sudo apt -y update
     $ sudo apt install g++ libomp-dev
```

Comment:
Please consider removing `$`. 
`&&` would let developers to copy and use this these commands right away.
```suggestion
    sudo apt -y update && sudo apt install -y g++ libomp-dev
```

Reply:
Yes, I was thinking the same about putting in a single line with the `&&`  :smile: